server/index.ts

```diff
@@ -32,6 +37,10 @@ contributionsRouter.get("/:id", getContribution(services));
 contributionsRouter.post("/", addContribution(services));
 app.use("/contributions", contributionsRouter);
 
+const twitterRouter = express.Router();
+twitterRouter.post("/verify", verify(services));
```

Comment:
how does this work? does the user click a button on our website to verify after they've posted?

Reply:
Ideally this is part of the flow (similar to how Interdependence worked) before they officially post